manifests/gcp_marketplace/guide.md

```diff
@@ -87,5 +87,10 @@ Reason:
 - Others created the cluster and deployed the instances for you.
 - You don't have corresponding permission to access it.
 
-Please ask admin to give your account `Project Editor` permission. It can be set
-from [IAM](https://console.cloud.google.com/iam-admin/iam).
+Actions:
+- Please ask admin to find out the Google Service Account used to create the cluster and then add your account as its `Service Account User` via [Service accounts](https://console.cloud.google.com/iam-admin/serviceaccounts). From the list table, check the
+service account, click the `Info Panel`, you will find a button `Add member` and add it
+as `Service Account User`. The Google Service Account is [Compute Engine default service account](https://cloud.google.com/compute/docs/access/service-accounts) if you didn't set it when creating the cluster.
```

Comment:
You can use this link: https://cloud.google.com/compute/docs/access/service-accounts#default_service_account

it is closer to what user needs to read